index.html

```diff
@@ -513,29 +513,140 @@ <h3>Assert Claim</h3>
+          </dd>
+        <dt>Motivations</dt>
+        <dd>
+          In many environments (such as order processing) information such
+          as a payer's address, citizenship, or age need to be
+          automatically verified in order to complete the transaction. 
+          
+          It MUST be possible to verify in an automated fashion.
+        </dd>
+        <dt>Needs</dt>
+        <dd>
+          <uref>F.2</uref>, <uref>C.1</uref>, <uref>E.2</uref>, <uref>R.1
+          </uref>,
+          <uref>F.5</uref>, <uref>H.2</uref>, <uref>C.6</uref>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h3>Validate Claim</h3>
+      <dl class="dl-horizontal">
+        <dt>Requirement</dt>
+        <dd>
+          It MUST be possible for a <a>verifier</a> to validate that the a
+          given claim is appropriate for particular use. That is, given the
+          credential is authentic and current, the verifier must then
+          interpret the claims in the context of their own 'business rules'
+          about which parties are acceptable authorities and which
+          cryptographic mechanisms are acceptable for different situations.
+          <ol>
+            <li>Is the issuer someone we know?</li>
+            <li>Are the claims made by this issuer appropriate for that
+            particular issuer's authority?</li>
+            <li>Is the holder's presentation an appropriate use of the
+            credential?</li>
+          </ol>
+          The first and second question will be answered either because the
+          verifier already knows the public identifiers for the issuers it
+          recognizes for specific claims, or there is a discovery mechanism
+          whereby verifiers can find appropriate issuers given their
+          business requirements. 
+          
+          The business rules may allow the use of certain credentials for
+          specific situations even when they are expired. For example a
+          suspended professional license may be accepted by an employer as
+          evidence of past experience without treating it as a current
+          qualification for roles that require that license. For this
+          reason, currency (as part of verification) only deals with the
+          status of the credential as maintained by the Issuer. Timeliness
+          is the domain of validation.
+
+          It's important to note that the bounds of authority for a given
+          issuer are specific claim types made by them and relied upon by
+          others. While the DMV might be an acceptable authority for an
+          individual's driving privilege, it should probably not be treated
+          as authoritative for an individual's hair color or current
+          address, even though DMVs regularly include such claims in
+          today's driver's licenses.
+
+          Unlike verification, validation cannot be guaranteed to be
```

Comment:
This misses an aspect of validation that can be automated, such as validating that the VC is conformant with the data model, has terms defined in a context, or complies with a schema.

Reply:
Perhaps distinguishing between real world understanding of the credential data vs machine based 'validation' is useful

Reply:
Some validation can be automated. I would certainly expect issuer lists and expiry rules to be fully automated by most verifiers. But there are use cases where validation cannot be solved in an automated way, e.g., when you need a live, real-time human to confirm a visual match of data in a credential and a physical specimen (whether human or materiel).

@decentralgabe I think you're still conflating verification and validation. Verification only checks authenticity and currency. Validation checks any business rules that might apply given a statement known to be authentic and current.

It is a requirement that verification be automatable. However, we have to support those validation use cases that cannot be.

Reply:
No further discussion. Marking resolved.